src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,42 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final int DATE_INDEX = 0;
+    private static final int NAME_INDEX = 1;
+    private static final int HOURS_INDEX = 2;
+    private static final int PER_HOUR_INDEX = 3;
+    private static final String WHITESPACE = " ";
+    private final DateTimeFormatter format = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder salaryInfo = new StringBuilder("Report for period "
+                + dateFrom + " - " + dateTo);
+
+        for (String name : names) {
+            int salary = 0;
+
+            for (String eachData : data) {
+                String[] info = eachData.split(WHITESPACE);
```

Comment:
we use constants to add more context to static values. So we need to explain why we need this value using names.
WHITESPACE - means the same as " " for us. Here it has a separation function, so we can call it like this
```suggestion
                String[] info = eachData.split(SEPARATOR);
```